raiden/network/rpc/client.py

```diff
@@ -780,52 +780,22 @@ def settled(self):
     def close(self, our_address, first_transfer, second_transfer):
```

Comment:
Now the order of the arguments is important, I think we should rename `first_transfer` to `their_transfer` and `second_transfer` to `our_transfer` to reflect the smart contracts function arguments.

Also, it's a programming error to call `close` without `their_transfer` and informing `our_transfer`, I think it should `raise ValueError`.